spec.html

```diff
@@ -31383,7 +31381,7 @@ <h1>Array.prototype.entries ( )</h1>
         <h1>Array.prototype.every ( _callbackfn_ [ , _thisArg_ ] )</h1>
         <emu-note>
           <p>_callbackfn_ should be a function that accepts three arguments and returns a value that is coercible to the Boolean value *true* or *false*. `every` calls _callbackfn_ once for each element present in the array, in ascending order, until it finds one where _callbackfn_ returns *false*. If such an element is found, `every` immediately returns *false*. Otherwise, if _callbackfn_ returned *true* for all elements, `every` will return *true*. _callbackfn_ is called only for elements of the array which actually exist; it is not called for missing elements of the array.</p>
-          <p>If a _thisArg_ parameter is provided, it will be used as the *this* value for each invocation of _callbackfn_. If it is not provided, *undefined* is used instead.</p>
+          <p>_thisArg_ parameter is used as the *this* value for each invocation of _callbackfn_.</p>
```

Comment:
I feel that the non-normative text here was more clear before the change. @ljharb what do you think?

Reply:
You might be right; it's a bit tricky given that callback might be in sloppy mode.

Reply:
Thanks for pointing it out, I've reverted all non-normative changes.